lib/Agrammon/Web/APIRoutes.pm6

```diff
@@ -80,9 +80,14 @@ sub rest-api-routes (Str $schema, Agrammon::Web::Service $ws) is export {
                         :language(~$language), :format(~$accept), :print-only(~$print-only),
                         :include-filters($include-filters eq 'true'), :all-filters($all-filters eq 'true')
                     );
-                    content ~$accept, supply {
-                        emit $results.encode('utf8');
-                    };
+                    if ~$accept eq 'application/json' {
```

Comment:
`$accept` header will be a `Str` and doesn't need coercing.

Reply:
Can still drop the `~` here too

Reply:
Overlooked and now dropped.